### PR TITLE
Fix bug 1366259: In latest activity tooltip, beautify FTL translations

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

### CODE REVIEW

File: pontoon/base/static/js/fluent_interface.js [hunk 1]
```diff
@@ -348,7 +348,7 @@ var Pontoon = (function (my) {
 $(function() {
 
   // Ignore editing for anonymous users
-  if (!Pontoon.user.id) {
+  if (Pontoon && Pontoon.user && !Pontoon.user.id) {
```
Comment (mathjazz) [inline pontoon/base/static/js/fluent_interface.js]: The `Pontoon` object is only available in translate view. Now that we run this code elsewhere, we need an additional check.

File: pontoon/base/static/js/table.js [hunk 2]
```diff
@@ -23,8 +23,12 @@ var date_formatter = new Intl.DateTimeFormat('en-GB', {
 $('body').on('mouseenter', '.latest-activity .latest time', function() {
   var $element = $(this);
 
+  // Translation Preview (needed for FTL)
+  var translationData = $element.data('translation'),
+      translationPreview = Pontoon.fluent.getSimplePreview({string: translationData}, translationData, {format: 'ftl'});
+
```
Comment (mathjazz) [inline pontoon/base/static/js/table.js]: Retrieving translation format is expensive ATM, so we assume all translations are in the FTL format. That way `getSimplePreview` will return simplified previews of FTL translations and fallback to the default presentation of all other translations.

File: pontoon/teams/templates/teams/team.html [hunk 3]
```diff
@@ -128,5 +128,7 @@
 {% endblock %}
 
 {% block extend_js %}
+  <!-- TODO: integrate into django-pipeline -->
```
Comment (jotes) [inline pontoon/teams/templates/teams/team.html]: Did you consider Webpack?

File: pontoon/base/static/js/fluent_interface.js (hunk 1 above)
Comment (adngdb) [inline pontoon/base/static/js/fluent_interface.js]: This should be ``if (!(Pontoon && Pontoon.user && Pontoon.user.id)) {``.

**Comment by adngdb** [comment]
@mathjazz Friendly reminder that this is r+'ed. :) Do you want to merge it? Is it blocked on anything?

**Comment by mathjazz** [comment]
I know, sorry... :) This is really ugly. In particular, I'd like to avoid enabling the FTL scripts on so many pages just to run this simple function.

I'm working on bug 1362072 now, as part of which I'll try to come up with a backend solution for rendering FTL strings, which will be useful in the translate view as well.

Closing.

**Pull request closed without merging**